src/main/app/forms/models/whatDocuments.ts

```diff
@@ -0,0 +1,41 @@
+import { MaxLength, ValidateIf, IsDefined } from 'class-validator'
+import { IsNotBlank } from 'app/forms/validation/validators/isNotBlank'
+import { Serializable } from 'models/serializable'
+
+export class ValidationErrors {
+  static readonly OTHER_DOCUMENTS_TOO_LONG: string = 'You’ve entered too many characters'
+  static readonly OTHER_DOCUMENTS_REQUIRED: string = 'Enter a description'
+}
+
+export class WhatDocuments implements Serializable<WhatDocuments> {
+
+  types?: string[]
+
+  @ValidateIf(o => o.types.indexOf('other') !== -1)
```

Comment:
is `other` a enum? Why don't you use const?

Reply:
Updated to use the type